src/App.tsx

```diff
@@ -310,27 +309,19 @@ function App() {
 
   return (
     <div className='container-fluid px-0'>
-      <nav className='navbar navbar-expand-lg navbar-light bg-light px-2'>
+      <nav className='navbar fixed-top navbar-expand-lg navbar-light bg-light px-2' style={{top:50}}>
```

Comment:
Let's move this style into the style sheet, instead of inlining it.

Reply:
added the .navbar-custom-style